system-parachains/bridge-hub-paseo/src/lib.rs

```diff
@@ -0,0 +1,1021 @@
+};
+
+#[cfg(any(feature = "std", test))]
+pub use sp_runtime::BuildStorage;
+
+// Polkadot imports
+use polkadot_runtime_common::{BlockHashCount, SlowAdjustingFeeUpdate};
+use paseo_runtime_constants::system_parachain::{ASSET_HUB_ID, BRIDGE_HUB_ID};
+
+use weights::{BlockExecutionWeight, ExtrinsicBaseWeight, RocksDbWeight};
+
+use parachains_common::{AccountId, Balance, BlockNumber, Hash, Header, Nonce, Signature};
+pub use system_parachains_constants::SLOT_DURATION;
+
+use system_parachains_constants::{
+    paseo::{consensus::*, currency::*, fee::WeightToFee},
+    AVERAGE_ON_INITIALIZE_RATIO, HOURS, MAXIMUM_BLOCK_WEIGHT, NORMAL_DISPATCH_RATIO,
+};
+
+// XCM Imports
+use xcm::prelude::*;
+
+/// The address format for describing accounts.
+pub type Address = MultiAddress<AccountId, ()>;
+
+/// Block type as expected by this runtime.
+pub type Block = generic::Block<Header, UncheckedExtrinsic>;
+
+/// A Block signed with a Justification
+pub type SignedBlock = generic::SignedBlock<Block>;
+
+/// BlockId type as expected by this runtime.
+pub type BlockId = generic::BlockId<Block>;
+
+/// The SignedExtension to the basic transaction logic.
+pub type SignedExtra = (
+    frame_system::CheckNonZeroSender<Runtime>,
+    frame_system::CheckSpecVersion<Runtime>,
+    frame_system::CheckTxVersion<Runtime>,
+    frame_system::CheckGenesis<Runtime>,
+    frame_system::CheckEra<Runtime>,
+    frame_system::CheckNonce<Runtime>,
+    frame_system::CheckWeight<Runtime>,
+    pallet_transaction_payment::ChargeTransactionPayment<Runtime>,
+    frame_metadata_hash_extension::CheckMetadataHash<Runtime>,
+);
+
+/// Unchecked extrinsic type as expected by this runtime.
+pub type UncheckedExtrinsic =
+generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
+
+parameter_types! {
+	pub DmpQueueName: &'static str = "DmpQueue";
+}
+
+/// Migrations to apply on runtime upgrade.
+pub type Migrations = (
+    // unreleased
+    frame_support::migrations::RemovePallet<DmpQueueName, RocksDbWeight>,
+    cumulus_pallet_xcmp_queue::migration::v4::MigrationToV4<Runtime>,
+    snowbridge_pallet_system::migration::v0::InitializeOnUpgrade<
+        Runtime,
+        ConstU32<BRIDGE_HUB_ID>,
+        ConstU32<ASSET_HUB_ID>,
+    >,
+    pallet_collator_selection::migration::v2::MigrationToV2<Runtime>,
+    // permanent
+    pallet_xcm::migration::MigrateToLatestXcmVersion<Runtime>,
+);
+
+/// Executive: handles dispatch to the various modules.
+pub type Executive = frame_executive::Executive<
+    Runtime,
+    Block,
+    frame_system::ChainContext<Runtime>,
+    Runtime,
+    AllPalletsWithSystem,
+    Migrations,
+>;
+
+impl_opaque_keys! {
+	pub struct SessionKeys {
+		pub aura: Aura,
+	}
+}
+
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+    spec_name: create_runtime_str!("bridge-hub-polkadot"),
+    impl_name: create_runtime_str!("bridge-hub-polkadot"),
+    authoring_version: 1,
+    spec_version: 1_001_002,
```

Comment:
@claravanstaden i have just realized about this. Did you forget upgrading the spec version to 1_001_005 or the code is effectively running under 1_001_002?